language/examples/document-summarization/summarization_with_docai.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>Could you change <code>%pip</code> to <code>! pip</code>  just to keep it consistent with other notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
This was a linter suggestion for the Notebook. I think that we should prefer using `%pip` instead of `!pip`

https://stackoverflow.com/a/76035586/6216983